src/Utilities/axiosInterceptors.js

```diff
@@ -9,18 +9,16 @@ const axiosInstance = axios.create();
 export function errorInterceptor(err) {
 
     if (!axios.isCancel(err)) {
+        const { response } = { ...err };
 
-        const { response, isAxiosError } = { ...err };
-
-        if (response && isAxiosError) {
+        if (response && err.isAxiosError) {
```

Comment:
For my understanding, what was the issue here?

Reply:
In the new version of axios

```js
const { response, isAxiosError } = { ...err }
console.log(isAxiosError); // prints "undefined"
console.log(err.isAxiosError); // prints "true"
```

It seems like by copying the object the destructuring and direct property access behave differently.